Maddori.Apple/Maddori.Apple/Screen/MyReflection/MyReflection/MyReflectionViewController.swift

```diff
@@ -113,7 +113,9 @@ extension MyReflectionViewController: UICollectionViewDelegate {
     }
     
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        self.navigationController?.pushViewController(MyReflectionDetailViewController(), animated: true)
+        guard let reflectionId = totalReflection[indexPath.item].id,
+        let reflectionName = totalReflection[indexPath.item].reflectionName else { return }
```

Comment:
p3;

```suggestion
        guard let reflectionId = totalReflection[indexPath.item].id,
                let reflectionName = totalReflection[indexPath.item].reflectionName else { return }
```

라인 plz.... 🙏

Reply:
[179601c](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/153/commits/179601c7737d7116ca04d2810eef05a64d85fe08)

하하 넵넵 감사합니다 '매' 토리 애쁠